frontend-react/src/components/SenderModeBanner.tsx

```diff
@@ -13,26 +13,9 @@ const isNotActive = (val: string): boolean => {
 
 // TODO (#5892): RSSender should be given attributes and constructor to be used as a Newable resource
 class RSSender {}
-const SenderAPI: API = {
-    resource: RSSender,
-    baseUrl: "/api/settings/organizations",
-    endpoints: new Map<string, Endpoint>([
-        [
-            "list",
-            {
-                url: "/:org/senders",
-                methods: ["GET"],
-            },
-        ],
-        [
-            "detail",
-            {
-                url: "/:org/senders/:sender",
-                methods: ["GET"],
-            },
-        ],
-    ]),
-};
+const SenderAPI = new API(RSSender, "/api/settings/organizations");
+SenderAPI.addEndpoint("list", "/:org/senders", ["GET"]);
+SenderAPI.addEndpoint("detail", "/:org/senders/:sender", ["GET"]);
```

Comment:
Modified the new API tooling to be cleaner to set up; hand writing objects is lame.